packages/hoppscotch-app/components/app/PaneLayout.vue

```diff
@@ -56,5 +64,62 @@ const SIDEBAR = useSetting("SIDEBAR")
 
 const slots = useSlots()
 
+const props = defineProps({
+  layoutId: {
+    type: String,
+    default: null,
+  },
+})
+
+type PaneEvent = {
+  max: number
+  min: number
+  size: number
+}
+
+const PANE_SIDEBAR_SIZE = ref(25)
+const PANE_MAIN_SIZE = ref(75)
+const PANE_MAIN_TOP_SIZE = ref(45)
+const PANE_MAIN_BOTTOM_SIZE = ref(65)
+const PANE_STORAGE_KEY = `${props.layoutId}-app-pane-layout`
+
+if (!COLUMN_LAYOUT) {
```

Comment:
`COLUMN_LAYOUT` is a reactive value. You have to access the value from `.value` property here. Change this to `COLUMN_LAYOUT.value`